src/watch/migrate.rs

```diff
@@ -39,6 +39,11 @@ impl Migrator {
         mut input: UnboundedReceiver<ExecutionOrder>,
         matcher: Arc<Watcher>,
     ) {
+        // Run initial dev-mode migration on startup
+        if let Err(e) = self.migration_apply_dev_mode().await {
+            print::error!("{e}");
+        }
```

Comment:
question: is there a way to just poke the `input` here so that we don't have to keep this bit aligned with the side effects below?

Reply:
Oh, that was my first attempt 👍 good question! It's just a bit tricky to rearrange the ownership of the sender `tx` here, so I just made another explicit call; the side effect could become a reused function if it grows larger in the future.

Reply:
Sure! Now fixed to use a do..while loop.